.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!--
+    Thanks for your contribution to the Open 3D Engine documentation! Before creating your pull
+    request, we ask you to sign off on our submission checklist to ensure that your PR comes through
+    quickly. Our reviewers' role is to make sure that your submission follows these best practices.
```

Comment:
Wondering if we should add a caveat that this is intended for non-trivial pull requests. Something like:
```suggestion
    Thanks for your contribution to the Open 3D Engine documentation! Before creating your pull
    request, we ask you to sign off on our submission checklist to ensure that your PR comes through
    quickly. Our reviewers' role is to make sure that all non-trivial submissions follow these best practices.
```